Source/CareerLog/CareerLogDto.cs

```diff
@@ -26,6 +26,8 @@ internal class CareerLogDto
         public double constructionFees;
         public double otherFees;
         public double fundsGainMult;
+        public double Reputation;
```

Comment:
Uhh, inconsistent casing strikes yet again

Reply:
Alright. One class used camelCase, the other Pascal case as far as I can see. I think that things are consistent. Can you please suggest alt names if not.